### ISSUE TITLE
Issue: FlexPaper (Standard) Viewer / Transition effect while opening a HTML(4) document in fullscreen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#312]
```
Transition effect while opening a HTML(4) document in fullscreen.
The width of the "flexpaper_viewer_container" container is changing before the 
document is shown in fullscreen mode.

This transitional effect is especially obvious when using an absolute position 
for the "flexpaper_viewer_container" container. 


Relevant Settings:
RenderingOrder : 'html'
```

Original issue reported on code.google.com by `mentur...@gmail.com` on 11 Oct 2012 at 2:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 15 Oct 2012 at 9:35
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting this! 
fixed in upcoming build 2.02
```

Original comment by `erik.eng...@devaldi.com` on 17 Oct 2012 at 11:35
- Changed state: **Fixed**
